src/gpgpu-sim/shader.cc

```diff
@@ -4639,6 +4670,82 @@ void simt_core_cluster::icnt_inject_request_packet(class mem_fetch *mf) {
                 mf->size());
 }
 
```

Comment:
I guess you can factor out the stat++ part and reuse it with icnt_inject_request_packet

Reply:
Factored out into a method `update_icnt_stats`.